dev/io.openliberty.org.apache.myfaces.4.0_fat/test-applications/ProgrammaticFaceletTests.war/src/io/openliberty/org/apache/faces40/fat/programmaticfacelettests/BasicViewFacelet.java

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package io.openliberty.org.apache.faces40.fat.programmaticfacelettests;
+
+import java.io.IOException;
+import java.util.List;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.faces.annotation.View;
+import jakarta.faces.component.UIComponent;
+import jakarta.faces.component.UIOutput;
+import jakarta.faces.component.html.HtmlBody;
+import jakarta.faces.component.html.HtmlForm;
+import jakarta.faces.component.html.HtmlHead;
+import jakarta.faces.component.html.HtmlOutputText;
+import jakarta.faces.context.FacesContext;
+import jakarta.faces.view.facelets.Facelet;
+
+/**
+ *
+ */
+@View("/BasicView.xhtml")
```

Comment:
This works, but can you escape some of the text? 
```
<html xmlns="http://www.w3.org/1999/xhtml" xmlns:h="http://xmlns.jcp.org/jsf/html">
Hello World!
</html>
```

`output.getAttributes().put("escape", false);`

Reply:
@pnicolucci  or should we leave it as is?   We might need to go to the community since myfaces doesn't escape but mojarra does.

Reply:
Update -- I was slightly incorrect above. MyFaces escapes special characters, so the html elements are visible on the page. Mojarra doesn't escape, so they don't face this issue. This should likely be brought up as a spec issue.

Reply:
Talked to Paul -- leave things as is. We can follow up on this later as I think this escape issue needs to resolved by the spec.